modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -54,23 +57,25 @@ const outcome = (grid: Structs.RowCells[], cursor: Optional<SugarElement>): Tabl
   cursor
 });
 
+const findEditableCursorPosition = (rows: Structs.RowCells[]) =>
+  Arr.findMap(rows, (row) =>
+    Arr.findMap(row.cells, (cell) => {
+      const elem = cell.element;
+      return Optionals.someIf(isEditable(elem), elem);
+    })
+  );
+
 const elementFromGrid = (grid: Structs.RowCells[], row: number, column: number) => {
   const rows = GridRow.extractGridDetails(grid).rows;
-  return findIn(rows, row, column).orThunk(() =>
-    findIn(rows, 0, 0)
-  );
+  return Optional.from(rows[row]?.cells[column]?.element)
+    .filter(isEditable)
+    // Fallback to the first valid position in the table
+    .orThunk(() => findEditableCursorPosition(rows));
```

Comment:
Do we normally have to put more heuristics in to try to put the selection somewhere _close_ to where it was previously, instead of just in the first content editable cell?

Reply:
This will try and place it close to where it was previously (which is what row and column is here). However, after that it's always just defaulted to the first viable position. Changing that is outside the scope of this PR so if you'd like to see that changed discuss it with Product/log a Jira please.